.github/workflows/build_and_test.yml

```diff
@@ -366,7 +366,7 @@ jobs:
         #   See also https://github.com/sphinx-doc/sphinx/issues/7551.
         # Jinja2 3.0.0+ causes error when building with Sphinx.
         #   See also https://issues.apache.org/jira/browse/SPARK-35375.
-        python3.6 -m pip install flake8 pydata_sphinx_theme mypy numpydoc 'jinja2<3.0.0'
+        python3.6 -m pip install flake8 pydata_sphinx_theme mypy numpydoc 'black==19.10b0' 'jinja2<3.0.0'
```

Comment:
Just a question, why specify this version?

Reply:
Otherwise it changes the formatting too much so we decided to pin the version and bump up manually ... Actually I think now is good time to bump up the version. @itholic can you try upgrade and see if it works?

Reply:
Upgrading the version changes the formatting as Hyukjin mentioned, but they look better to read than the old code. (And it's not break the `pycodestyle` as well)

I think it's good time to use the latest version of `black` now.

Let me remove the specified versions.

Reply:
Can you still pin the version (to the latest one)? when another version is released, the CI will be broken.